data/tlg0018/tlg001/tlg0018.tlg001.1st1k-eng1.xml

```diff
@@ -0,0 +1,1060 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">The Works of Philo Judaeus</title>
+<sponsor>University of Leipzig</sponsor>
+<funder>European Social Fund Saxony</funder>
+<principal xml:id="GRC">Gregory Crane</principal>
+<respStmt>
+<persName xml:id="Jouve">Jouve</persName>
+<resp>Digital Conversion and edition</resp>
+</respStmt>
+<respStmt>
+<persName>Gregory Crane</persName>
+<resp>Digital conversion and editing</resp>
```

Comment:
Digital conversion implies Lace work, but this file says Jouve. So I would recommend simply Digital editing in this case.

Reply:
Well, it implies OCR work or some sort of conversion. I used tesseract or OCR text from Hathi now.